spread.yaml

```diff
@@ -226,7 +226,7 @@ backends:
                   image: ubuntu-20.04-64
                   username: ubuntu
                   password: ubuntu
-                  flags: [virtio]
```

Comment:
so we don't get virtio anymore? or was this just always being ignored in our current GCE spread runs?

Reply:
This section is only about qemu images and this particular syntax was never part of the official spread so removing it is fine.

Reply:
TBH I have a local branch of spread that just adds those flags unconditionally, so it's interesting there is an unofficial flags field. May I ask if this in your private fork? If so, which branch?

Reply:
oh right this is just for qemu not for google. I would have preferred to keep it for the cases where we do a local qemu spread run since virtio is much faster since I myself was using this branch of spread: https://github.com/snapcore/spread/pull/96 however now I see that there is a better branch of spread which does virtio always: https://github.com/snapcore/spread/pull/131 so I will probably switch to that one instead

Reply:
Yeah, sorry for the mess, the original PR that added `flags: [virto]` never got accepted. Fwiw, testing on 21.04 with qemu seems to use virtio or is so fast that I don't notice the difference. I have not looked into the details but the need vanished for me.

Reply:
I can keep it for now of course if you prefer that.

Reply:
I think it's fine to remove, we should probably switch our personal forks to using your newer PR if it's still necessary, I suppose I haven't actually checked on 21.04 about qemu using virtio automatically or if there is still a significant speed difference between using it and not using it

Reply:
FWIW, I remember that some images (18.04 amd64?) did not boot at all with virtio enabled as the kernel was apparently missing required driver support..

Also I would not mind if we share what's in our personal spread forks and come up with a version that's tailored to our daily use cases.

Reply:
Given that noone approved the PR with the removal I added it back and just added a TODO, hope this is mergable now :)